Documentation/workflow/OfficalAndDailyBuilds.md

```diff
@@ -12,11 +12,36 @@ and you can expect to see new versions roughly three months.   However it is als
 Team publishes **daily builds** of all sorts of packages including those built by the CoreCLR and CoreFX 
 repositories.  You can see what is available from
 
- * <https://dotnet.myget.org/gallery/dotnet-core>, and in particular you can see the builds of CoreCLR at 
- * <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Runtime.CoreCLR>.   
- 
+ * <https://dotnet.myget.org/gallery/dotnet-core>, and in particular you can see the builds of 
+ * CoreCLR at <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Runtime.CoreCLR>
+ * NETCore.App at <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App>
+
 Thus if your goal is just to get the latest bug fixes and features, you don't need to build CoreCLR yourself you 
-can simply add <https://dotnet.myget.org/F/dotnet-core/api/v3/index.json> to your Nuget Feed list. 
+can simply add <https://dotnet.myget.org/F/dotnet-core/api/v3/index.json> to your Nuget Feed list and set the
+`RuntimeFrameworkVersion` in your project file to a `Microsoft.NETCore.App` version. You need to restore
+and publish your application so it includes the runtime (`self-contained`).
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+ <packageSources>
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json"/>
+ </packageSources>
+</configuration>
+```
+```
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>2.0.0-preview2-*/RuntimeFrameworkVersion>
+  </PropertyGroup>
+</Project>
+```
+```
+$ dotnet restore -r linux-x64
+$ dotnet publish -r linux-x64 --self-contained true
```

Comment:
@eerhardt If you have a large solution where some projects have an RID set. Suppose you want to use a different RID. If you specify that as a property to msbuild, then projects which didn't have an rid set (-> publishing not selfcontained), will now publish self-contained. Is that right? Isn't that an issue?